packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -332,6 +332,20 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
     markNeedsTextLayout();
   }
 
+  /// Character used for obscuring text if [obscureText] is true.
+  ///
+  /// Cannot be null, and must have a length of exactly one.
+  String get obscuringCharacter => _obscuringCharacter;
+  String _obscuringCharacter;
+  set obscuringCharacter(String value) {
+    if (_obscuringCharacter == value) {
+      return;
+    }
+    assert(value != null && value.length == 1);
+    _obscuringCharacter = value;
+    markNeedsLayout();
```

Comment:
The `markNeedsSemanticsUpdate()` is probably still needed. See:
https://github.com/flutter/flutter/blob/987607a23138979deff4c51bd8f98f68e3b4525b/packages/flutter/lib/src/rendering/editable.dart#L1228-L1241

Reply:
It will be done as part of the layout. See
https://github.com/flutter/flutter/blob/f582246af659e8bcbba5588b19464ba1a119ffed/packages/flutter/lib/src/rendering/object.dart#L1631